hack/after-build/update-generated-docs.sh

```diff
@@ -39,8 +39,10 @@ shopt -u dotglob
 
 kube::util::gen-analytics "${KUBE_ROOT}"
 
+KUBE_GIT_UPSTREAM="${KUBE_GIT_UPSTREAM:-upstream}"
```

Comment:
David do you want me to fix this here?  I don't have a use case outside of docs for this yet, although I could see us int he future.